addons/locales.sh

```diff
@@ -18,9 +15,17 @@ then
     download_script STATIC fetch_lib
 fi
 
+# Check for errors + debug code and abort if something isn't right
+# 1 = ON
+# 0 = OFF
+DEBUG=0
+debug_mode
+
 # Must be root
 root_check
```

Comment:
6. We could move that header to the fetch_lib file, if we want to shorten the header
And btw: is debug_mode really needed, if you can just run `bash -x /path/to/script` ?

Reply:
Honestly I never used debug mode, but I don't know if some of the several thousands of users actually use it. It doesn't hurt to leave it as is imo.

Reply:
Okay